services-in-london/src/App.js

```diff
@@ -1,12 +1,78 @@
 import React, { Component } from 'react';
 import './App.css';
 import Area from './area';
-import Organization from './organization';
+import Type from './type';
```

Comment:
Although `Type` is ok to use in Javascript (see [reserved words](http://www.javascripter.net/faq/reserved.htm)), it is in many other languages, so best to use a more specific word like `OrganisationType` or something like that